### ISSUE TITLE
add support for typescript "go to source definition"

### ISSUE THREAD

**Issue by christophsturm** [sveltejs/language-tools#1890]
### Description

typescript now has a great new feature , go to source definition, but it does not yet work in svelte files in visual studio code.
see here: 

https://github.com/microsoft/TypeScript/issues/49003
and
https://github.com/microsoft/TypeScript/issues/49003#issuecomment-1439939552

### Proposed solution

it would be great if this could be added to the svelte vscode plugin. If someone could point me to a good starting point i could try to implement it.

**Comment by jasonlyu123** [comment]
I don't think this is suitable for a first-time contributor. Yes. There is an API for it, but we're using a lower-level API, so we can't access that API. We have to copy a lot of the logic from TypeScript to make it happen. For that reason, This won't happen any time soon.
